src/Cylinder.cpp

```diff
@@ -0,0 +1,181 @@
+#include "Cylinder.hpp"
+
+#include <fstream>
+
+template class Cylinder<2U>;
+template class Cylinder<3U>;
+
+template <unsigned int dim>
+Cylinder<dim>::Cylinder(const std::string &mesh_file_name_,
+                        const unsigned int &degree_velocity_,
+                        const unsigned int &degree_pressure_, const double &T_,
+                        const double &deltat_, const double &U_m_,
+                        const SolverOptions &solver_options_,
+                        const bool &compute_lift_drag_)
+    : NavierStokes<dim>(mesh_file_name_, degree_velocity_, degree_pressure_, T_,
+                        deltat_, solver_options_),
+      U_m(U_m_),
+      compute_lift_drag(compute_lift_drag_),
+      zero_function(dim + 1) {
+  NavierStokes<dim>::ro = 1.0;   //[kg/m^3].
+  NavierStokes<dim>::nu = 1e-3;  //[m^2/s].
+  NavierStokes<dim>::initial_conditions =
```

Comment:
You don't really need to specify the scope `NavierStokes<dim>::` in classes derived from `NavierStokes` (such as this one). It is only needed if there's a name clash (e.g. you have a member called `ro` in both `NavierStokes<dim>` and `Cylinder<dim>`), but this is not the case here.